model/transformers/examples/qa/train_data.sh

```diff
@@ -0,0 +1,66 @@
+cat >> train.json << EOF
+{
+    "data":[
+       {
+          "title":"University_of_Notre_Dame",
+          "paragraphs":[
+             {
+                "context":"Architecturally, the school has a Catholic character. Atop the Main Building's gold dome is a golden statue of the Virgin Mary. Immediately in front of the Main Building and facing it, is a copper statue of Christ with arms upraised with the legend \"Venite Ad Me Omnes\". Next to the Main Building is the Basilica of the Sacred Heart. Immediately behind the basilica is the Grotto, a Marian place of prayer and reflection. It is a replica of the grotto at Lourdes, France where the Virgin Mary reputedly appeared to Saint Bernadette Soubirous in 1858. At the end of the main drive (and in a direct line that connects through 3 statues and the Gold Dome), is a simple, modern stone statue of Mary.",
```

Comment:
It looks like this is from a book. We can't use something that might be copyrighted (and I'd assume is in this case, unless you find out otherwise)

Reply:
Oh! This is from SQuAD dataset distributed under CC-BY-SA-4.0 license. Is that problematic? If so, I can make up something to write here.

Reply:
That's perfect! We just need to document that somewhere